strike/schema.go

```diff
@@ -82,7 +82,7 @@ type Body_structure struct {
 
 	// URI of the next API in the chain
 	//
-	NextActionHandler string `json:"nextActionHandler,omitempty"`
+	NextActionHandler string `json:"nextActionHandlerURL,omitempty"`
```

Comment:
I've conveyed this to the frontend once they'll change I'll merge so that the code doesn't break. 
Rest LGTM.